connectors/sources/salesforce.py

```diff
@@ -773,6 +1076,104 @@ def _format_address(self, address):
         ]
         return ", ".join([a for a in address_fields if a])
 
+    def _format_time_for_sort(self, time_body_pair):
+        if time_body_pair[0] is None:
+            return ""
+
+        return time_body_pair[0]
```

Comment:
You can inline this function into the method that uses it, if it's the only place, it simplifies things a bit IMO

Reply:
Made it an in-line lambda 👍🏻